src/popup/components/InputAmount.vue

```diff
@@ -11,6 +11,7 @@
     :message="$attrs['message'] || errors.first('amount')"
     class="input-amount"
     type="number"
+    inputmode="decimal"
```

Comment:
In my opinion this should be moved into InputField component and be based on input type.